jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     '^.+\\.tsx?$': 'ts-jest',
   },
   testEnvironment: 'node',
-  testTimeout: 15000,
+  testTimeout: 30000,
```

Comment:
😬  lol this has to do with my high volume 500 million dollar integration test trade for swap + add. i can limit the tests to 100-200 million if preferred.